index.html

```diff
@@ -0,0 +1,255 @@
+          <li class="bar"></li>
+        </ul>
+        <ul class="nav-items">
+          <li>
+            <a href="#" class="page1">
+              <img class="logo" src="./img/logo.png" alt="logo image"
+            /></a>
+          </li>
+          <li class="header-text1"><a href="#" class="page1">Home</a></li>
+          <li><a href="#" class="about-page">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#partner">Sponsor</a></li>
+          <li class="logo2">UN climate change</li>
+        </ul>
+      </div>
+    </header>
+
+    <!-- page 1-->
+    <div class="page1">
+      <section class="home">
+        <div class="home-title">
+        <h3 class="sub-title">Sharm el-Sheikh Climate Change Conference - November 2022</h3>
+        <h1 class="main-title">
+          What did COP 27 achieve and what needs to happen next?
+        </h1>
+        <p class="paragraph1">
+          Establishing a dedicated fund for loss and damage,
+          Maintaining a clear intention to keep 1.5°C within reach, Holding businesses and institutions to account, 
+          Mobilizing more financial support for developing countries, and Making the pivot toward implementation.
+        </p>
+        </div>
+        <h2 class="date">6 Nov - 20 Nov 2022</h2>
+        <p class="address">
+          @ UN Climate Change
+        </p>
+      </section>
+
+      <!-- main-program -->
+      <section id="program" class="main-program">
+        <h2 class="program-title">Main Program</h2>
+        <hr class="line1" />
+        <div class="program">
+          <div class="program-tab">
+            <img src="./img/program_icon_01.png" alt="icon" />
+            <h3 class="program-text1">Lecture</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_02.png" alt="icon" />
+            <h3 class="program-text1">CC Exhibition</h3>
+            <p class="program-text2">
+              Lorem, ipsum amet consectetur adipisicing elit. Consectetur harum
+              velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_03.png" alt="icon" />
+            <h3 class="program-text1">Forum</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_04.png" alt="icon" />
+            <h3 class="program-text1">Workshop</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
```

Comment:
[REQUIRED] Kindly ensure that the text colour and dot-like background match the template.

![image](https://user-images.githubusercontent.com/62938735/212386249-a144060e-6cbd-4995-94cd-6b3b4b489986.png)

![image](https://user-images.githubusercontent.com/62938735/212386400-ecf4a8b3-48e2-419f-9beb-0cf5154d7594.png)